### ISSUE TITLE
New flags should start at beginning of a selected range with the same size

### ISSUE THREAD

**Issue by aedrax** [rizinorg/cutter#2935]
**Is your feature request related to a problem? Please describe.**

When selecting multiple addresses in the disassembly widget, adding a new flag chooses the last address as the starting address with a default size of 1. 

**Describe the solution you'd like**

When selecting a range of addresses, adding a new flag should default to the starting address, and the default size should be the length of the entire selected range

**Comment by karliss** [comment]
Slightly related to #1708 also #1709

**Comment by aedrax** [comment]
I agree that it is very much related to #1709 and this should probably be implemented in the same fix. The only additional thing this suggests is the extra details of the selection when applying to a flag. Should I close this ticket as a duplicate?
